Audio/AudioDriverCSCore.cs

```diff
@@ -122,8 +122,11 @@ private void Output_Stopped(object sender, PlaybackStoppedEventArgs e)
 
         public void Dispose()
         {
-            if ( aout != null )
+            if (aout != null)
+            {
+                aout.Stop();
                 aout.Dispose();
+            }
```

Comment:
Should probably nullify `aout` after disposal to prevent `aout.Stop()` from throwing if this were to get called multiple times.